src/components/filterPatterns/popover/index.tsx

```diff
@@ -111,12 +111,14 @@ export const PopoverFilter: FC<PopoverFilterProps> = ({
                     alignItems="center"
                   >
                     {Icon ? <Icon h="xs" w="xs" mr="xs" /> : null}
-                    {[
-                      displayValue ? appliedLabel ?? label : label,
-                      displayValue,
-                    ]
-                      .filter(Boolean)
-                      .join(': ')}
+                    <chakra.span overflow="hidden" textOverflow="ellipsis">
```

Comment:
The `textOverflow` property doesn't work with flexbox directly. Adding additional wrapper for the text  can display again our lovely ellipsis